ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -40,10 +40,6 @@ tests:
     env:
       TEST_REQUIRES_SSH: "yes"
       TEST_TYPE: upgrade-conformance
-    pre:
-    - ref: ipi-install-hosted-loki
```

Comment:
why is loki going away?

Reply:
it's part of the default workflow, we overrode pre when we added this and copied in, now we should fall back to the normal workflow which already includes this as our pre section is going away.